utils/Makefile.am

```diff
@@ -22,3 +22,24 @@ hcitop_LDADD = \
 	@LIBBSD_LIBS@ \
 	@NCURSES_LIBS@
 endif
+
+if ENABLE_BASH_COMPLETION
+bashcompdir = @BASH_COMPLETION_DIR@
+bashcomp_DATA = bash-completion/bluealsa
+
+EXTRA_DIST = $(bashcomp_DATA)
+
+SYMLINKS = \
+	a2dpconf \
+	bluealsa-aplay \
+	bluealsa-cli \
+	bluealsa-rfcomm \
+	hcitop
+
+install-data-hook:
+	cd $(DESTDIR)$(bashcompdir) && \
+	for link in $(SYMLINKS); do ln -sf bluealsa $$link; done
```

Comment:
Add `AC_PROG_LN_S` and `AC_PROG_MKDIR_P` to the `configure.ac` and before making links, create destination directory (user can specify non-existing location). Instead of `mkdir` and `ln` use $(MKDIR_P) and $(LN_S). Also, I'm not sure whether `ln -f` is a good idea - if the link is already there maybe it should be left alone, but I'm not sure to be honest.

Reply:
Other projects that "own"  link targets delete them first with rm -f before re-creating with ln -s. I guess that's because ln -f is not portable. So I think its OK for the bluealsa installer to forcibly remove these links. The bluez-alsa project "owns" the links just as much as it "owns"  the bluealsa executable and bluealsa bash completion script, and it has no reservations about overwriting them.

I'll revise the makefile to also rm/ln sequence instead of ln -sf, using the autotools macros as you request.